gen/enum_test.go

```diff
@@ -501,6 +502,69 @@ func TestEnumLabelValid(t *testing.T) {
 	}
 }
 
+func TestEnumStrictMarshalText(t *testing.T) {
+	tests := []struct {
+		testName   string
+		enumStrict tems.EnumMarshalStrict
+		hasError   bool
+	}{
+		{"marshalText success",
+			1,
+			false,
+		},
+		{
+			"marshalText unknown value",
+			5,
+			true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(string(tt.testName), func(t *testing.T) {
```

Comment:
testName is already a string. the cast is not needed.
```suggestion
		t.Run(tt.testName, func(t *testing.T) {
```